static/js/util/validation.js

```diff
@@ -87,6 +87,26 @@ export function educationValidation(profile) {
   }
 }
 
+export function educationUiValidation(profile, ui) {
+  if (profile.education === undefined) {
+    profile = Object.assign({}, profile, {
+      education: []
+    });
+  }
+
+  let errors = {};
+  for (let {value, label} of EDUCATION_LEVELS) {
+    let items = profile.education.filter(education => education.degree_name === value);
+    if (ui.educationDegreeInclusions[value] && items.length === 0) {
+      errors[`education_${value}_required`] = `${label} is required if switch is set`;
+    }
+    if (!ui.educationDegreeInclusions[value] && items.length > 0) {
+      errors[`education_${value}_required`] = `${label} switch is off but there is still education history in card`;
+    }
+  }
+  return errors;
+}
```

Comment:
So when they turn the switch off but didn't delete, do you want them to turn it back on and delete? 

Can we just delete when they turn the switch off? Maybe with the confirmation message that Alice was working on?